vendor/init.go

```diff
@@ -66,22 +83,65 @@ var files = []file{
 	},
 }
 
+func addChrome() (err error) {
+	var chromeFile file
+	chromeFile.name = chromeFilename
+	chromeFile.browser = true
+	ctx := context.Background()
```

Comment:
Nit: Pass in the context to the function.

context.Background() should generally only be called in function main.